vpp-manager/vpp_runner.go

```diff
@@ -45,8 +45,9 @@ import (
 )
 
 var (
-	fakeNextHopIP4    = net.ParseIP("0.0.0.0")
-	fakeNextHopIP6    = net.ParseIP("::")
+	fakeNextHopIP4s   = map[string]net.IP{"eth0": net.ParseIP("0.0.0.0")}
+	fakeNextHopIP6s   = map[string]net.IP{"eth0": net.ParseIP("::")}
+	priority          = 1
```

Comment:
We should not store it, but probably compute it when calling `AddRoute()` as we have the list at this point as well